tests/hacker.test.js

```diff
@@ -562,11 +597,13 @@ describe("PATCH update one hacker", function() {
                 agent.close();
                 return done(error);
             }
+            let app = TeamHacker0.application;
+            app.other.gender = "Other";
```

Comment:
Nit: Maybe send application: {other: {gender: "Other"}}} ?

Reply:
I agree. We shouldn't mutate the test data.